python/hail/expr/tests.py

```diff
@@ -1005,13 +1005,13 @@ def test_array_methods(self):
         self.assertEqual(hl.eval_expr(hl.flatmap(lambda x: hl.range(0, x), [1, 2, 3])), [0, 0, 1, 0, 1, 2])
 
     def test_bool_r_ops(self):
-        self.assertTrue(hl.eval_expr(hl.capture(True) & True))
-        self.assertTrue(hl.eval_expr(True & hl.capture(True)))
-        self.assertTrue(hl.eval_expr(hl.capture(False) | True))
-        self.assertTrue(hl.eval_expr(True | hl.capture(False)))
+        self.assertTrue(hl.eval_expr(hl.lit(True) & True))
```

Comment:
Do we prefer hl.bool(True) in this case when we clearly know the type?

Reply:
I'm not sure. I just did find/replace here. Happy to change it if you think that's appropriate

Reply:
I don't have a strong opinion.  There were a few cases of `hl.lit([1, 2, 3])` and `hl.array([1, 2, 3])` and I thought I'd raise the question.  I'm happy with it as-is.